.travis.yml

```diff
@@ -9,11 +9,13 @@ addons:
       - postgresql-10-postgis-2.5
 services:
   - postgresql
+before_install:
+  - sudo apt-get install -y sqlite3
+  - sudo apt install -y libgdal-dev libproj-dev libproj-dev
+  - sudo apt install -y libsqlite3-dev spatialite-bin libsqlite3-mod-spatialite
 install:
   - pip install -r requirements.txt
   - pip install codecov
-before_script:
```

Comment:
@BarisSari - don't we need to install postgis any more?

Reply:
We are installing the package on line 9:  **postgresql-10-postgis-2.5** . However, we need to create extension for each new database. Therefore, I moved the code in initialise()